packages/blob/src/index.ts

```diff
@@ -1,8 +1,8 @@
 import type { PutCommandOptions } from './put';
 import { createPutMethod } from './put';
 
-// expose generic BlobError
-export { BlobError } from './helpers';
+// expose generic BlobError and download url util
+export { BlobError, getDownloadURL } from './helpers';
```

Comment:
Let's never uppercase fully URL, let's only use camelCaseUrl everywhere (public and private APIs)